front/app/components/UI/QuillEditor/altTextToImagesModule.tsx

```diff
@@ -0,0 +1,149 @@
+// Implementation inspired by:
+// https://quilljs.com/guides/cloning-medium-with-parchment/
+// https://gist.github.com/tranduongms1/584d43ec7d8ddeab458f087adbeef950
+
+import Quill, { QuillOptionsStatic } from 'quill';
+
+const Module = Quill.import('core/module');
+const BlockEmbed = Quill.import('blots/block/embed');
+
+export const attributes = ['alt', 'width', 'height', 'style'];
+
+// Create a custom ImageBlot that allows us to add alt text to the image
+export class ImageBlot extends BlockEmbed {
+  static blotName = 'image';
+  // Instead of using the default img tag, we are using a div where we can add an input field for the alt text
+  static tagName = ['div'];
+
+  static create(value: { alt: string; src: string } | string) {
+    const node = super.create();
```

Comment:
If you could add a comment here too, that would be great. (Already indicating what this node will be for example.)